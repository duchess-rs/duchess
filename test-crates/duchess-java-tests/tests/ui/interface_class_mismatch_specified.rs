duchess::java_package! {
    package exceptions;

    public interface ThrowExceptions { } //~ ERROR: error in class
}

fn main() {}
