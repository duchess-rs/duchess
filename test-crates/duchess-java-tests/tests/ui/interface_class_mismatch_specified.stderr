error: error in class `exceptions.ThrowExceptions`: class `exceptions.ThrowExceptions` should be type `Class`
 --> tests/ui/interface_class_mismatch_specified.rs:4:5
  |
4 |     public interface ThrowExceptions { }
  |     ^^^^^^

error: aborting due to 1 previous error

